tsconfig.json

```diff
@@ -5,14 +5,15 @@
     "downlevelIteration": true,
     "esModuleInterop": true,
     "experimentalDecorators": true,
-    "lib": ["es2020", "ES2021.Intl"],
-    "noImplicitAny": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["ES2020", "ES2021.Intl"],
+    "module": "CommonJS",
```

Comment:
I tried setting this to `node16` since [that's what's recommended](https://github.com/tsconfig/bases/blob/main/bases/node14.json) but it broke the resolution for a handful of dependencies that ship with both CJS/ESM builds. I love this ecosystem 😭

Reply:
you might want to build out some integration tests that run the compiled bin, maybe with `--help`?, to make sure that it works on all versions of node

Reply:
does this count? https://github.com/readmeio/rdme/blob/198d4c42c8eca7a723e2ba8e7842cf87f8f08485/__tests__/bin.test.ts#L8

Reply:
oh sick, i forgot we had those.